partner_aging/wizard/customer.py

```diff
@@ -0,0 +1,178 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Ursa Information Systems <http://www.ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo import tools
+from odoo import api, fields, models, _
+
+
+class PartnerAgingDate(models.TransientModel):
+    _name = "partner.aging.date"
+
+    age_date = fields.Datetime("Aging Date",
+                               required=True,
+                               default=lambda self: fields.Datetime.now())
+
+    @api.multi
+    def open_partner_aging(self):
+        ctx = self._context.copy()
+        ctx.update({'age_date': self.age_date})
+
+        customer_aging = self.env['partner.aging.customer.ad']
+        customer_aging.execute_aging_query(age_date=self.age_date)
+
+        return {
+            'name': _('Customer Aging'),
+            'view_type': 'form',
+            'view_mode': 'tree,form',
+            'res_model': 'partner.aging.customer.ad',
+            'type': 'ir.actions.act_window',
+            'domain': [('total', '!=', 0)],
+            'context': ctx,
+        }
+
+
+class AccountAgingCustomerAD(models.Model):
+    _name = 'partner.aging.customer.ad'
```

Comment:
This name does not match the class name. Also, I'm kind of unclear on what this model does by the naming. I don't have a suggestion though, but would be happy to provide one if you elaborate on what you were going for?